artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -755,6 +771,23 @@ public void initChannel(Channel channel) throws Exception {
       logger.debug("Started {} Netty Connector version {} to {}:{}", connectorType, TransportConstants.NETTY_VERSION, host, port);
    }
 
+   private String processSslPasswordProperty(String password, String codecClass) {
```

Comment:
Instead of using `ConfigurationHelper` you could simply use `PasswordMaskingUtil` (which is what `ConfigurationHelper` does eventually), e.g.:
```java
   private String resolveMask(String password, String codecClass) {
      try {
         return PasswordMaskingUtil.resolveMask(password, codecClass);
      } catch (Exception e) {
         throw new RuntimeException(e);
      }
   }
```

Reply:
Oh, yes. Much better! I wanted to reuse the `ConfigurationHelper` password logic since the extra Boolean logic was already handled there. Without that `useMaskedPassword` that reasoning and need goes right out the window.

Updated and pushed.